src/main/scala/com/fortysevendeg/mvessel/statement/Statement.scala

```diff
@@ -1,21 +1,21 @@
-package com.fortysevendeg.android.sqlite.statement
+package com.fortysevendeg.mvessel.statement
 
-import java.sql._
+import java.sql.{SQLException, SQLWarning, Statement => SQLStatement, ResultSet => SQLResultSet}
 
 import android.database.Cursor
-import com.fortysevendeg.android.sqlite.logging.{AndroidLogWrapper, LogWrapper}
-import com.fortysevendeg.android.sqlite.resultset.SQLDroidResultSet
-import com.fortysevendeg.android.sqlite.{SQLDroidConnection, SQLDroidDatabase, WrapperNotSupported, _}
-import StatementInfo._
+import com.fortysevendeg.mvessel.logging.{AndroidLogWrapper, LogWrapper}
+import com.fortysevendeg.mvessel.resultset.ResultSet
+import com.fortysevendeg.mvessel.statement.StatementInfo._
+import com.fortysevendeg.mvessel.{Connection, Database, WrapperNotSupported, javaNull}
 
 import scala.collection.mutable
 import scala.util.{Failure, Success, Try}
 
-class SQLDroidStatement(
-  sqlDroidConnection: SQLDroidConnection,
+class Statement(
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Number of methods in class exceeds 30](https://www.codacy.com/app/47deg/scala-sqlite-droid/file/822213828/issues/source?bid=2442814&fileBranchId=2752966#l14)